consensus/src/chained_bft/chained_bft_consensus_provider.rs

```diff
@@ -136,29 +117,22 @@ impl ChainedBftProvider {
         );
         InitialSetup {
             author,
+            epoch: 0,
```

Comment:
Perhaps add a TODO here about supporting epoch transition for those unaware of the change.